internal/pkg/cmd/flags.go

```diff
@@ -396,24 +398,18 @@ func AddTypeFlag(cmd *cobra.Command) {
 }
 
 func AddKeyFormatFlag(cmd *cobra.Command) {
-	arr := []string{"string", "avro", "jsonschema", "protobuf"}
-	str := utils.ArrayToCommaDelimitedString(arr, "or")
-
-	cmd.Flags().String("key-format", "string", fmt.Sprintf("Format of message key as %s. Note that schema references are not supported for Avro.", str))
+	cmd.Flags().String("key-format", serializationFormats[0], fmt.Sprintf("Format of message key as %s. Note that schema references are not supported for Avro.", utils.ArrayToCommaDelimitedString(serializationFormats, "or")))
```

Comment:
i suggest using "string" instead of serializationFormats[0] here. serializationFormats[0] is at the top of this file, a little hard to read